tools/local-network-setup/setup-linux-environment.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+pathToOtNode=$(pwd)
+numberOfNodes=4
```

Comment:
Should I also set `12` here and in the MacOS script?

Reply:
yes

Reply:
Done in https://github.com/OriginTrail/ot-node/pull/2922/commits/5523a798cf116877e59f6b5b00087f45d2e16905.